### ISSUE TITLE
fork/exec /var/task/autospotting: permission denied: PathError null

### ISSUE THREAD

**Issue by vdoodala** [cristim/autospotting#262]
## Issue type ##

- Bug Report

## Build number #

```
fork/exec /var/task/autospotting: permission denied: PathError
null
```

## Environment ##

- AWS region
- Type of environment: (VPC, EC2Classic or DefaultVPC)
- Anonymized launch configuration: 

```
Region us-west-2
```

## Summary ##

```
The autosptting is not working for ASG. Please guide me how to proceed further. 
```

**Comment by cristim** [comment]
Is this happening with the official binaries?

**Comment by vdoodala** [comment]
I used the same module https://github.com/cristim/autospotting/tree/master/terraform and changed parameters in variables.tf like bucket name etc.. Lambda function everything is built... I can see this error  fork/exec /var/task/autospotting: permission denied: PathError
null in cloudwatch logs.

**Comment by cristim** [comment]
Sorry but I can't support any custom built binaries.

**Comment by vdoodala** [comment]
I am not using any custom build binaries... I just set the names for these variables...
variable "lambda_zipname" {
  description = "Name of the archive"
  default     = "autospotting.zip"
}

variable "lambda_s3_bucket" {
  description = "Bucket which the archive is stored in"
  default     = "<bucket I created>"
}

variable "lambda_s3_key" {
  description = "Key in S3 under which the archive is stored"
  default     = "autospotting.zip"
}

**Comment by cristim** [comment]
Try getting a ZIP file from [here](https://cloudprowess.s3.amazonaws.com/index.html), upload it to your S3 bucket and update your stack to point to it.

**Comment by cristim** [comment]
@vdoodala did you get it work?

**Comment by vdoodala** [comment]
yes, it's working ... thank you for help. Please close the issue.
